### PR TITLE
Add Mac M1 sidecar build

### PR DESCRIPTION
**Pull request by mpfz0r**
## Notes for Reviewers

- [ ] The commit history must be preserved - please use the rebase-merge or standard merge option instead of squash-merge
- [ ] Sync up with the author before merging

### CODE REVIEW

**Comment by mpfz0r** [comment]
FYI: @kodjo-anipah successfully tested this already :+1:

File: Makefile [hunk 1]
```diff
@@ -49,37 +49,37 @@ solaris-sigar-patch:
...
 
 build-windows: install-goversioninfo ## Build sidecar binary for Windows
 	@mkdir -p build/$(COLLECTOR_VERSION)/windows/amd64
 	$(GOVERSIONINFO_BIN) -64 -product-version="$(COLLECTOR_VERSION)-$(COLLECTOR_REVISION)" -ver-major="$(COLLECTOR_VERSION_MAJOR)" -product-ver-minor="$(COLLECTOR_VERSION_MINOR)" -product-ver-patch="$(COLLECTOR_VERSION_PATCH)" -product-ver-build="$(COLLECTOR_REVISION)" -file-version="$(COLLECTOR_VERSION)-$(COLLECTOR_REVISION)" -ver-major="$(COLLECTOR_VERSION_MAJOR)" -ver-minor="$(COLLECTOR_VERSION_MINOR)" -ver-patch="$(COLLECTOR_VERSION_PATCH)" -ver-build="$(COLLECTOR_REVISION)" -o resource_windows.syso
-	GOOS=windows GOARCH=amd64 CGO_ENABLED=1 CC=x86_64-w64-mingw32-gcc $(GO) build $(BUILD_OPTS) -pkgdir $(GOPATH)/go_win -v -i -o build/$(COLLECTOR_VERSION)/windows/amd64/graylog-sidecar.exe
+	GOOS=windows GOARCH=amd64 CGO_ENABLED=1 CC=x86_64-w64-mingw32-gcc $(GO) build $(BUILD_OPTS) -pkgdir $(GOPATH)/go_win -v -o build/$(COLLECTOR_VERSION)/windows/amd64/graylog-sidecar.exe
 
 build-windows32: install-goversioninfo ## Build sidecar binary for Windows 32bit
-	@mkdir -p build/$(COLLECTOR_VERSION)/windows/386
+	@"mkdir -p build/$(COLLECTOR_VERSION)/windows/386
```
Comment (thll) [inline Makefile:80]:
```suggestion
	@mkdir -p build/$(COLLECTOR_VERSION)/windows/386
```

File: Makefile [hunk 2]
```diff
@@ -49,37 +49,37 @@ solaris-sigar-patch:
...
-	GOOS=linux GOARCH=386 $(GO) build $(BUILD_OPTS) -pkgdir $(GOPATH)/go_linux32  -v -i -o build/$(COLLECTOR_VERSION)/linux/386/graylog-sidecar
+	GOOS=linux GOARCH=386 $(GO) build $(BUILD_OPTS) -pkgdir $(GOPATH)/go_linux32  -v -o build/$(COLLECTOR_VERSION)/linux/386/graylog-sidecar
 
 build-darwin: ## Build sidecar binary for OSX
 	@mkdir -p build/$(COLLECTOR_VERSION)/darwin/amd64
-	GOOS=darwin GOARCH=amd64 $(GO) build $(BUILD_OPTS) -pkgdir $(GOPATH)/go_darwin -v -i -o build/$(COLLECTOR_VERSION)/darwin/amd64/graylog-sidecar
+	GOOS=darwin GOARCH=amd64 $(GO) build $(BUILD_OPTS) -pkgdir $(GOPATH)/go_darwin -v -o build/$(COLLECTOR_VERSION)/darwin/amd64/graylog-sidecar
 
 build-darwin-arm64: ## Build sidecar binary for OSX
 	@mkdir -p build/$(COLLECTOR_VERSION)/darwin/amd64
```
Comment (thll) [inline Makefile:67]:
```suggestion
	@mkdir -p build/$(COLLECTOR_VERSION)/darwin/arm64
```

Comment (mpfz0r) [reply Makefile:67]: ah. apparently this mkdir isn't really needed at all. but :shrug:

**Review by thll — APPROVED** [review]
Looks good.

Didn't test the binary itself, but it builds nicely :smile:

File: Makefile (hunk 2 above)
Comment (thll) [reply Makefile]: yeah, I've also noticed that 😂 . now it's at least consistent...

**Pull request merged**